src/operations/common_functions.ts

```diff
@@ -42,7 +42,7 @@ export function indexInformation(
 
   // Did the user destroy the topology
   if (getTopology(db).isDestroyed())
-    return callback(new MongoDriverError('topology was destroyed'));
+    return callback(new MongoTopologyClosedError('Topology was destroyed'));
```

Comment:
```suggestion
    return callback(new MongoTopologyClosedError('Topology is destroyed'));
```
If we're going to update messages to present tense, can do this one as well.